### ISSUE TITLE
I want to use it with Pythonista (iPhone).

### ISSUE THREAD

**Issue by KohnoseLami** [dashingsoft/pyarmor#430]
I provide some useful scripts in Pythonista but they are not compatible ...
I tried to run something encrypted on my Mac, but they said it was incompatible and I couldn't.
If this problem can be solved immediately, could you please deal with it?

**Comment by jondy** [comment]
Please obfuscate the scripts with platform `darwin.aarch64`, for example,
```
pyarmor obfuscate --platform darwin.aarch64 foo.py
```
If pyarmor is not the latest version v.6.6.0,  use `darwin.arm64` instead.

**Comment by KohnoseLami** [comment]
```python
dlopen(/private/var/mobile/Containers/Shared/AppGroup/D17BC1C4-E089-45BD-9B78-1B82F31B7E58/Pythonista3/Documents/dist_1/dist/pytransform/_pytransform.dylib, 0x0006): code signature in (/private/var/mobile/Containers/Shared/AppGroup/D17BC1C4-E089-45BD-9B78-1B82F31B7E58/Pythonista3/Documents/dist_1/dist/pytransform/_pytransform.dylib) not valid for use in process: mapped file has no cdhash, completely unsigned? Code has to be at least ad-hoc signed.
```

Unable to run with following error ...

**Comment by jondy** [comment]
Try to sign it with ad-hoc sign by this command:
```
codesign -f -s - /path/to/_pytransform.dylib
```

**Comment by KohnoseLami** [comment]
codesign seems to require a developer account to run.
If xcode works well, it seems that dylib can be signed with a normal account, but do you think there are any other good ways?

It is very difficult to sign.

**Comment by jondy** [comment]
I upload a singed one here
https://pyarmor.dashingsoft.com/downloads/temp/_pytransform.dylib

Please download it in the current path, and obfuscate the scripts like this
```
pyarmor obfuscate --platform ./_pytransfrom.dylib foo.py
```

Check it work or not.

**Comment by KohnoseLami** [comment]
```
dlopen(/private/var/mobile/Containers/Shared/AppGroup/D17BC1C4-E089-45BD-9B78-1B82F31B7E58/Pythonista3/Documents/dist_3/dist/pytransform/_pytransform.dylib, 0x0006): code signature invalid (errno=1) sliceOffset=0x00000000, codeBlobOffset=0x00119B40, codeBlobSize=0x00006AA0 for '/private/var/mobile/Containers/Shared/AppGroup/D17BC1C4-E089-45BD-9B78-1B82F31B7E58/Pythonista3/Documents/dist_3/dist/pytransform/_pytransform.dylib'
```

Authentication appears to be invalid: (

**Comment by jondy** [comment]
I think the runtime library `dist/pytransform/_pytransform.dylib` need to be signed again after `pyarmor obfuscate`, no more idea about code sign, please refer to apple document or google to find more solutions.

**Comment by KohnoseLami** [comment]
I used my Apple Developer account to sign the Apple Distribution, but the signature is no longer valid and cannot be used.
I think this is a problem with _pytransform.dylib. Can you solve it?

**Comment by johnlundberg** [comment]
I think I might have the same issue. We sign the dlls we ship with our product, and it worked great until version 6.3
Before that we could sign _pytransform.dll before the obfuscation.

But now if we sign _pytransform.dll before obfuscation it is no longer a valid signature since the dll is modified during the obfuscation.

I tried using --with-license outer but the dll still seems to be modified, I am guessing with what previously was saved in pytransform.key?

If we sign the _pytransform.dll after the obfuscation, I get this stack trace
Traceback (most recent call last):
  File "<executable.py>", line 3, in <module>
  File "<frozen executable>", line 70, in <module>
  File "<frozen executable>", line 62, in protect_pytransform
  File "<frozen executable>", line 54, in check_lib_pytransform


I tried setting restrict mode to 0, but with same result. Any way to turn off this check or make signing work?
